api/tests/Feature/PoolCandidateUpdateTest.php

```diff
@@ -310,30 +310,30 @@ public function testCanQueryForNotifications()
                 /** @lang GraphQL */
                 '
                 query myNotifications {
```

Comment:
Maybe we should move these tests into the new file?

Reply:
Yeah, I can do that. Were you thinking of moving it to the `NotificationTest` or creating a new `PoolCandidateStatusChangedNotifcationTest` file?

Reply:
I was thinking they belong in `NotificationTest`.

Reply:
Sounds good 👍 Moved in [consolidate notification tests](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9694/commits/8a13a0a5cb7126067f4e3d734da1dd5f437b10ac)